homeassistant/config_entries.py

```diff
@@ -694,6 +694,15 @@ def async_update_entry(self, entry, *, data=_UNDEF, options=_UNDEF):
         if integration is None:
             raise data_entry_flow.UnknownHandler
 
+        try:
```

Comment:
Instead of this whole diff, we can just do

```python
try:
    integration.get_component()
except ImportError:
    raise data_entry_flow.UnknownHandler
```

Reply:
it never returns None?

Reply:
No, it will raise `ImportError` in that case